### DISCUSSION TITLE
Application initialization failed

### DISCUSSION THREAD

**Discussion by Chi379** [bullhorn/career-portal#537] [Q&A]
I've followed all the steps (I think) but I'm getting the following error: Application initialization failed, please check your app.json file: Http failure during parsing for https://domainaname.com/careers/app.js

Can anyone help?

**Comment by pbearne** [comment]
change domainanamecom to your domain

**Reply by Chi379** [reply]
Thanks - yes, I just put domainname.com in as a place holder as my site is not ready for the public just yet.
